modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -52,7 +52,7 @@ $ oc create -f volumesnapshotclass.yaml
 .volumesnapshot-dynamic.yaml
 [source,yaml]
 ----
-apiVersion: snapshot.storage.k8s.io
+apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshot
 metadata:
   name: mysnap
```

Comment:
@bobfuru 
I don't know why I cannot comment one following line about volumeSnapshotClassName setting, so I added here.
<1> The request for a particular class by the volume snapshot. If `volumeSnapshotClassName` is empty, then no snapshot is created.

I tested in OCP4.7, If `volumeSnapshotClassName` is empty and there is a default volumeSnapshotClass, it will add the default volumeSnapshotClass behind and create snapshot successfully, like default storageclass, please add a note here.

Reply:
Please verify that my rewording is correct here:

> <1> The request for a particular class by the volume snapshot. If the `volumeSnapshotClassName` setting is empty but there is a default volume snapshot class name, a snapshot is created. But if the field is empty and no default volume snapshot class name exists, then no snapshot is created.

Reply:
@bobfuru 
I need to double confirm with you that here empty mean we did not specify the `volumeSnapshotClassName` field. And in another case you specify like this `volumeSnapshotClassName: ""`, it will be denied by admission webhook. I suggest to update following, free free to modify to accord with doc spec :)

<1> The request for a particular class by the volume snapshot. If the `volumeSnapshotClassName` is absent and there is a default volume snapshot class, a snapshot will be created with the default volume snapshot class name. But if the field is absent and no default volume snapshot class exists, then no snapshot will be created.

Reply:
@duanwei33 Yes, you are correct that my intention for "empty" was to mean "absent". I've updated with your suggestion:

> If the `volumeSnapshotClassName` setting is absent and there is a default volume snapshot class, a snapshot is created with the default volume snapshot class name. But if the field is absent and no default volume snapshot class exists, then no snapshot is created.